[SC-150] Wyświetlanie pozycji studenta w rankingu

dmncp opened the pull request:
Dodałem wyróżnienie kolorem wiersza odwzorowującego pozycję zalogowanego studenta. Na razie oczywiście mockowa, zahardcode'owana pozycja

dmncp committed:
[SC-156] students ranking for teacher view

dmncp committed:
[SC-150] ranking table for student view

kwdrt reviewed (approved):
Działa!

I stylistycznie - wygląda super, nie wiem czamu ale bardzo mi się podoba. 

Odnośnie kodu też bez zastrzeżeń, tylko te same drobne uwagi co przy SC-156, tak samo nieblokujące

dmncp committed:
[SC-150] Merge remote-tracking branch 'origin/develop' into SC-150-wyswietlanie-pozycji-studenta-w-rankingu

# Conflicts:
#	frontend/src/components/general/Ranking/Ranking.js
#	frontend/src/components/general/Ranking/RankingStyle.js
#	frontend/src/components/general/Ranking/mockData.js
